src/hotspot/share/classfile/classLoader.cpp

```diff
@@ -242,6 +242,7 @@ ClassPathDirEntry::~ClassPathDirEntry() {
 }
 
 ClassFileStream* ClassPathDirEntry::open_stream(JavaThread* current, const char* name) {
+  // current is Thread::current().
```

Comment:
This is unnecessary

Reply:
Thanks. I remove this and update the comment as well.